LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Eddie Jaoude
+Copyright (c) 2021 - present Eddie Jaoude
```

Comment:
```suggestion
Copyright (c) 2021 - 2022 Eddie Jaoude
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix